src/substrate-lib/components/developerConsole.ts

```diff
@@ -0,0 +1,14 @@
+// This component will simply add utility functions to your developer console.
+import { useSubstrate } from '..';
+
+export default function DeveloperConsole (props) {
```

Comment:
It is better to avoid `export default`. Just export it

Reply:
`export default` is in the main branch, so that seems unrelated? I'm happy to  make the change if you want.